homeassistant/components/tibber/__init__.py

```diff
@@ -34,16 +77,21 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     return True
 
 
-async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+async def async_setup_entry(hass: HomeAssistant, entry: TibberConfigEntry) -> bool:
     """Set up a config entry."""
 
+    if AUTH_IMPLEMENTATION not in entry.data:
```

Comment:
Since this is migration code, let's add a code comment about when it was added and when it could be removed.